.travis.yml

```diff
@@ -39,9 +39,11 @@ env:
 
 addons:
   apt:
+    sources:
+      - sourceline: 'ppa:jblgf0/python'
     packages:
       - maven
-      - python3
+      - python3.9
```

Comment:
It would seem that this is all that is needed to get the desired version. Shouldn't then `python3` resolve to this version?

Reply:
No, there seems to be a pre-installed python 3.5 which led to the original problem and other failures during the testing of this PR. The scripts themselves must explicitly specify the exact python version.
And for other python3.9 to work correctly, the correct pip and distutils are needed.